src/screens/TokenSwap.js

```diff
@@ -363,11 +383,12 @@ const styles = StyleSheet.create({
   },
   screenContent: {
     flex: 1,
-    backgroundColor: COLORS.lowContrastDetail,
+    backgroundColor: '#fff',
```

Comment:
issue: We must use the `COLORS` object to centralize all colors in the app. This is a necessary step towards having dark mode in the future.

There are two more hardcoded colors in this file that should also centralize their values in `COLORS`.

Reply:
Done